internal/service/cloudbackupsnapshotexportjob/data_source_cloud_backup_snapshot_export_job.go

```diff
@@ -71,8 +71,9 @@ func DataSource() *schema.Resource {
 				Computed: true,
 			},
 			"err_msg": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:       schema.TypeString,
+				Computed:   true,
+				Deprecated: fmt.Sprintf(constant.DeprecationParamByVersion, "1.18.0"),
```

Comment:
isn't 1.18.0 too soon? It's basically our next release which might contain this change itself.

Reply:
You are right! Changed to 1.20.0